tests/Main.hs

```diff
@@ -365,15 +365,22 @@ sinkEntrySpec =
 loadEntrySpec :: SpecWith (Path Abs File)
 loadEntrySpec =
   context "when an entry is loaded" $
-    it "is there" $ \path -> property $ \m b s -> do
+    it "is there" $ \path -> property $ \m b s r -> do
       let vpath = deriveVacant path
       B.writeFile (toFilePath vpath) b
+      addUTCTime (-60) <$> liftIO getCurrentTime >>= setModificationTime vpath
```

Comment:
There are two things being checked:

1. If user wants restoration, he gets restoration.
2. If the user doesn't want restoration, he doesn't get restoration.

Normally, without calling setModificationTime, the modification time is the time when the file is created. By intentionally setting the time of the original file to be one minute ago, we can expect a large time difference between the two modification times if Point 2 is fulfilled.

Do you think Point 2 is unnecessary?